taskfiles/charts.yml

```diff
@@ -0,0 +1,129 @@
+version: '3'
+
+env:
+  # some of the tasks defined in this file are executed in containers
+  # see https://github.com/golang/go/issues/53640 for details on why
+  # 1.19.2 fails to deal with modules with nested repositories. Once
+  # the fix for #53640 is included in a release, either next minor for
+  # 1.19 or 1.20, this can be removed
+  GOFLAGS: -buildvcs=false
+  GOPRIVATE: github.com/redpanda-data/flux-controller-shim
+
+tasks:
+  lint:
```

Comment:
What would you think about instead consolidating this into the top level `lint`? Having a single entry point for `lint` and preventing duplication when possible will go a long way for maintainability. The existing golangcilint configuration will perform all the checks that staticcheck and gofumpt currently do so we can remove those.

Reply:
ok I will try